main.py

```diff
@@ -2,6 +2,31 @@
 
 from django.db.models import QuerySet
 
+from db.models import Actor, Genre
+
 
 def main() -> QuerySet:
-    pass
+    with open("data.txt", "r") as f:
+        lines = f.read().splitlines()
+
+    for line in lines:
+        data = line.split(" ")
+
+        kind_of_data = data[0]
+
+        if kind_of_data == "genre":
+            genre = data[1]
+            Genre.objects.create(name=genre)
+        else:
+            first_name, last_name = data[1], data[2]
+            Actor.objects.create(first_name=first_name, last_name=last_name)
```

Comment:
It's interesting choice, but I imagine how you wroted those file manually 😣 You should avoid additional work. Will be much more optimized just to create tuples with data inside a function. Thats what you should think about: to make it more optimized and to do less work)

Reply:
it was actually harder to use tuples instead of a file :)